troposphere/ecs.py

```diff
@@ -35,6 +65,9 @@ class Service(AWSObject):
         'DesiredCount': (positive_integer, False),
         'LoadBalancers': ([LoadBalancer], False),
         'Role': (basestring, False),
+        'PlacementConstraints': ([PlacementConstraint], False),
```

Comment:
I think the convention is for these props to be alphabetical. The `Placement...` ones should probably be moved above `Role`.